packages/content/src/Domain/Model/AuthorTrait.php

```diff
@@ -30,6 +30,11 @@ trait AuthorTrait
      */
     private $authored;
 
+    /**
+     * @var \DateTimeImmutable|null
+     */
+    private $lastModified;
```

Comment:
@alexander-schranz @martinlagler why did we add this here to the `AuthorInterface`?

In 2.6 we had a separate interface for this [LastModifiedBehavior](https://github.com/sulu/sulu/blob/2.6/src/Sulu/Component/Content/Document/Behavior/LastModifiedBehavior.php)